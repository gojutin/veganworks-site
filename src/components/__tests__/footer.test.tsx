import React from "react";
import renderer from "react-test-renderer";
import Footer from "../footer";
import "jest-styled-components";

describe("Footer", () => {
  it("matches snapshot", () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
