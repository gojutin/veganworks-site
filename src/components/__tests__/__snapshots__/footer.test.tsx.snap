// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer matches snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  background: #efefef;
  position: fixed;
  bottom: 0;
  left: 0;
  width: 100%;
  padding: 12px;
}

<footer
  className="c0"
>
  <div
    className=""
  >
    <a
      css="
        color: steelblue;
        cursor: pointer;
      "
      href="https://github.com/gojutin/gatsby-starter"
      rel="noopener noreferrer"
      target="_blank"
    >
      <img
        src="test-file-stub"
        width={30}
      />
    </a>
  </div>
</footer>
`;
